frame_id: "world_origin"

leader_msg: "mpc"
frame_publish: "utm_origin"

timer_main:
  rate: 2.0 # [Hz]

timer_pose:
  rate: 2.0 # [Hz]

center:
  x: 5.0 # [m]
  y: 5.0 # [m]
  z: 2.0 # [m]

dimensions:
  x: 80.0 # [m]
  y: 80.0 # [m]

angle: 0.0
lateral_scale: 1.0
side: 6.0
